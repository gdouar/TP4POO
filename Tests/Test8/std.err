Erreur : le dernier parametre doit etre le nom de fichier