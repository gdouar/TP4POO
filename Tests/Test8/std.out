Warning : parametre inconnu non pris en compte : -h
Warning : parametre inconnu non pris en compte : -W
Warning : parametre inconnu non pris en compte : test.dot